toDoApp/app/src/main/java/hieunguyen/com/todoapp/MainActivity.java

```diff
@@ -0,0 +1,181 @@
+package hieunguyen.com.todoapp;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    MyAdapter itemsAdapter;
+    ListView listView;
+    MainService dbService = new MainService(this);
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        itemsAdapter = new MyAdapter(this, dbService);
+
+        updateListHeader();
+
+        listView = (ListView) findViewById(R.id.lvItems);
+        listView.setAdapter(itemsAdapter);
+
+        // On tap, allow user to edit task
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                createEditDialog(position).show();
+            }
+        });
+
+        // On long hold, allow user to delete task
+        listView.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+            @Override
+            public boolean onItemLongClick(AdapterView<?> parent, View view, int position, long id) {
+                createDeleteDialog(position).show();
+                return true;
+            }
+        });
+
+        Button clearCompleted = (Button) findViewById(R.id.clearCompleted);
+        Button addTask = (Button) findViewById(R.id.addNew);
+
+        // Remove completed tasks
+        clearCompleted.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                dbService.clearCompleted();
+                itemsAdapter.setList(dbService.getDataFromDB());
+                updateListHeader();
+            }
+        });
+
+        // On tap, allow user to add task
+        addTask.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                createAddDialog().show();
+            }
+        });
+    }
+
+    /**
+     * Updates the header that says "X To Do"
+     */
+    public void updateListHeader() {
+        TextView header = (TextView) findViewById(R.id.header);
+        String count = itemsAdapter.getCount() == 0 ? "Nice! Nothing" : String.valueOf(itemsAdapter.getCount());
+        header.setText(count + " To Do");
```

Comment:
You should add "To do" to your strings.xml file
Use Alt + Enter in Android Studio to extract a string resource